docs/tutorials/tutorial_cjoyce.md

```diff
@@ -0,0 +1,48 @@
+# Tutorial: Enroll a new user
+
+
+Using this tutorial, you will add (`POST`) the details of a new [`user resource`](../api/user.md) to the service, enrolling a new user!
+
+This should take about **15 minutes** to complete.
+
+## Before you start
+
+1. Make sure you've completed the [Before you start a tutorial](../before-you-start-a-tutorial.md) topic. This will ensure your development system is running correctly!
+2. Start your local service if it isn't already running by using the following command:
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+## Let's get started
+
+
+1. Using the Postman app, set up a new request with these values:
+    * **Method**: POST
+    * **URL**: `{{base_url}}/users`
+        * When run locally for testing, the {base_url} is generally http://localhost:3000.
+    * **Headers**:
+        * `Content-Type: application/json`
+    * **Request body**:
+        Please reference the [user resource page](../api/user.md) for a guide to each property.
+
+        ```js
```

Comment:
Be sure to review the indented code examples in the built and published docs to make sure they align as you'd like them to. I think they render differently in the preview than they do in the final docs. If they're not indented correctly they can mess up the surrounding list elements.